ha/core/system_health/system_health.py

```diff
@@ -307,19 +307,85 @@ def publish_event(self, healthevent: HealthEvent, healthvalue: str= ""):
         self.producer.publish(str(healthevent))
         healthevent.node_id = node_id
 
+    # Placeholder function to detect if status of all pods is collected
+    # before "sys_health_bootstrap_timeout". Function , logic to be deleted
+    # if the requried data for the same is not going to be avilable from cluster.conf
+    def _check_num_pods(self):
```

Comment:
If the count of pods is known and not all pods status events are received, what will be the health status of cluster, site, rack? Unknown?

Reply:
This is a placeholder function and since we are not doing any health aggregation currently, cluster/site/rack status will not get changed due to it. But once health aggregation is added in HA, this will impact the cluster status

Reply:
On second thoughts, I think its better to remove the function call altogether and keep only a comment mentioning the same here for the current commit.

Reply:
Yes the functionality related to cluster cardinality can be handled in separate Jira. Removing this function for now is fine.